pytorch_blade/benchmark/TorchBench/common.py

```diff
@@ -0,0 +1,1313 @@
+            sys.stdout.write(f"{current_device:4} {mode:5} {current_name:34} ")
+            sys.stdout.flush()
+            for submod in itertools.chain([model], model.modules()):
+                assert not torchdynamo.utils.is_jit_model(submod)
+
+            torch.manual_seed(1337)
+            correct_result = model_iter_fn(
+                copy.deepcopy(model), torchdynamo.utils.clone_inputs(example_inputs)
+            )
+
+            torch.manual_seed(1337)
+            if current_name not in self.non_deterministic_models:
+                correct_rerun_result = model_iter_fn(
+                    copy.deepcopy(model), torchdynamo.utils.clone_inputs(example_inputs)
+                )
+                if not same(correct_result, correct_rerun_result):
+                    print("INCORRECT - Variation in Eager runs itself")
+                    if not skip_accuracy_check:
+                        return sys.exit(-1)
+
+            torch.manual_seed(1337)
+            torchdynamo.reset()
+
+            if experiment.func is cold_start_experiment:
+                results = []
+                results.append(experiment(model, example_inputs, optimize_ctx))
+                print(" ".join(map(str, results)))
+                return 0
+
+            try:
+                with optimize_ctx:
+                    new_result = model_iter_fn(model, example_inputs)
+            except Exception:
+                logging.exception("unhandled error")
+                print("ERROR")
+                return sys.exit(-1)
+            if current_name in self.non_deterministic_models:
+                # This model has non-deterministic output so we cant
+                # check correctness.
+                # TODO(jansel): submit upstream fix for this
+                pass
+            elif not same(correct_result, new_result, cos_similarity, tolerance):
+                print("INCORRECT")
+                if not skip_accuracy_check:
+                    return sys.exit(-1)
+            ok, total = Stats.reset_counters()
+            results = []
+
+            # run one more time to see if we reached a fixed point
+            with optimize_ctx:
+                model_iter_fn(model, example_inputs)
+            _, frames_second_pass = Stats.reset_counters()  # should be 0
+
+            if frames_second_pass > 0:
+                with optimize_ctx:
+                    model_iter_fn(model, example_inputs)
+                _, frames_third_pass = Stats.reset_counters()  # should be 0
+            else:
+                frames_third_pass = 0
+
+            if output_filename and "coverage" in output_filename:
+                results.append(
+                    f"{ok:3}/{total:3} +{frames_third_pass} frames {time.perf_counter()-t0:3.0f}s"
+                )
+
+            results.append(experiment(model, example_inputs))
+            print(" ".join(map(str, results)))
+
+
+def help(fn):
+    return fn.__doc__
+
+
+def parse_args():
```

Comment:
Nit: how about overriding the parse_args from `torchdynamo.benchmarks.common` directly? Such as writing the following codes in some files like bladebench.py
```python
import logging
from torchdynamo.benchmarks import common
from torchdynamo.benchmarks import torchbench

def parse_args():
    ...
    return args

if __name__ == "__main__":
    common.parse_args = parse_args
    logging.basicConfig(level=logging.WARNING)
    warnings.filterwarnings("ignore")
    commons.main(torchbench.TorchBenchmarkRunner(), torchbench.original_dir)
    
```

Reply:
And then call the TorchBench with the following command:

`python bladebench.py --speedup-blade-disc -d cuda --isolate --float32 --skip-accuracy-check 2>&1 | tee speedup_blade.log`